### ISSUE TITLE
 Compiliation error on Linux : Overload function confliction

### ISSUE THREAD

**Issue by AtlantixJJ** [torch/cutorch#790]
I am using Ubuntu 14.04 and CUDA 7.5. During my installation of torch, a bunch of errors is produced, which is listed (partly) here:

    THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
                function "min(int, int)"
                function "min(unsigned int, unsigned int)"
                function "min(int, unsigned int)"
                function "min(unsigned int, int)"
                function "min(long long, long long)"
                function "min(unsigned long long, unsigned long long)"
                function "min(long long, unsigned long long)"
                function "min(unsigned long long, long long)"
                function "min(float, float)"
                function "min(double, double)"
                function "min(float, double)"
                function "min(double, float)"
                argument types are: (unsigned long, unsigned long)

I think my problem is exactly the same as [this](https://github.com/torch/cutorch/issues/551) but the latest version of cutorch seems to produce this error, too.

I try `luarocks install cutorch` and clone cutorch repo and `luarocks make rocks/cutorch-1.0-0.rockspec` but both failed at the same place.

And then I tried to reset to the [old version](https://github.com/torch/cutorch/pull/548) that fixed this issue and then the compilation goes well.

But the problem is `install.sh` script in torch seems to force the latest version of packages and the installation of torch is still incomplete in my case.

In conclusion I think some PR broke the compilation again, any ideas?

Also may I ask for advices for installing torch with cutorch reset to old version ?

Thanks.

**Comment by soumith** [comment]
can you give a full log file of errors? I can fix this.

**Comment by AtlantixJJ** [comment]
The log is here. Hope it can help. Thanks.

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bff2_00000000-7_THCTensorMathReduceChar.cpp1.ii".
CMake Error at THC_generated_THCTensorMathReduceChar.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathReduceChar.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathReduceChar.cu.o] Error 1
make[2]: *** Waiting for unfinished jobs....
1 error detected in the compilation of "/tmp/tmpxft_0000bfea_00000000-7_THCTensorMathReduceShort.cpp1.ii".
1 error detected in the compilation of "/tmp/tmpxft_0000bfe3_00000000-7_THCTensorMathReduceByte.cpp1.ii".
CMake Error at THC_generated_THCTensorMathReduceShort.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathReduceShort.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathReduceShort.cu.o] Error 1
CMake Error at THC_generated_THCTensorMathReduceByte.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathReduceByte.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathReduceByte.cu.o] Error 1
/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bf3b_00000000-5_THCTensorMathCompareTByte.cpp4.ii".
CMake Error at THC_generated_THCTensorMathCompareTByte.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathCompareTByte.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathCompareTByte.cu.o] Error 1
/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bf7b_00000000-5_THCTensorIndex.cpp4.ii".
CMake Error at THC_generated_THCTensorIndex.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir//./THC_generated_THCTensorIndex.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/./THC_generated_THCTensorIndex.cu.o] Error 1
1 error detected in the compilation of "/tmp/tmpxft_0000bf2f_00000000-5_THCTensorMathCompareTChar.cpp4.ii".
1 error detected in the compilation of "/tmp/tmpxft_0000bf48_00000000-5_THCTensorMathCompareTShort.cpp4.ii".
CMake Error at THC_generated_THCTensorMathCompareTShort.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathCompareTShort.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathCompareTShort.cu.o] Error 1
CMake Error at THC_generated_THCTensorMathCompareTChar.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathCompareTChar.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathCompareTChar.cu.o] Error 1
/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bf39_00000000-5_THCTensorMathPointwiseByte.cpp4.ii".
1 error detected in the compilation of "/tmp/tmpxft_0000bf4a_00000000-5_THCTensorMathPointwiseShort.cpp4.ii".
CMake Error at THC_generated_THCTensorMathPointwiseByte.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathPointwiseByte.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathPointwiseByte.cu.o] Error 1
CMake Error at THC_generated_THCTensorMathPointwiseShort.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathPointwiseShort.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathPointwiseShort.cu.o] Error 1
1 error detected in the compilation of "/tmp/tmpxft_0000bf9d_00000000-5_THCTensorMathReduce.cpp4.ii".
1 error detected in the compilation of "/tmp/tmpxft_0000bf6e_00000000-5_THCTensorMathPointwiseChar.cpp4.ii".
CMake Error at THC_generated_THCTensorMathPointwiseChar.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathPointwiseChar.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMathPointwiseChar.cu.o] Error 1
CMake Error at THC_generated_THCTensorMathReduce.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir//./THC_generated_THCTensorMathReduce.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/./THC_generated_THCTensorMathReduce.cu.o] Error 1
/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bf9a_00000000-5_THCTensorMath2.cpp4.ii".
CMake Error at THC_generated_THCTensorMath2.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir//./THC_generated_THCTensorMath2.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/./THC_generated_THCTensorMath2.cu.o] Error 1
/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/generated/../THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bffd_00000000-5_THCTensorTopK.cpp4.ii".
CMake Error at THC_generated_THCTensorTopK.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir//./THC_generated_THCTensorTopK.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/./THC_generated_THCTensorTopK.cu.o] Error 1
1 error detected in the compilation of "/tmp/tmpxft_0000bfa3_00000000-5_THCTensorMaskedChar.cpp4.ii".
CMake Error at THC_generated_THCTensorMaskedChar.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMaskedChar.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMaskedChar.cu.o] Error 1
1 error detected in the compilation of "/tmp/tmpxft_0000bfc6_00000000-5_THCTensorRandom.cpp4.ii".
CMake Error at THC_generated_THCTensorRandom.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir//./THC_generated_THCTensorRandom.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/./THC_generated_THCTensorRandom.cu.o] Error 1
1 error detected in the compilation of "/tmp/tmpxft_0000bfdd_00000000-5_THCTensorMaskedShort.cpp4.ii".
CMake Error at THC_generated_THCTensorMaskedShort.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMaskedShort.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMaskedShort.cu.o] Error 1
1 error detected in the compilation of "/tmp/tmpxft_0000bfc3_00000000-5_THCTensorMaskedByte.cpp4.ii".
CMake Error at THC_generated_THCTensorMaskedByte.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMaskedByte.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/generated/./THC_generated_THCTensorMaskedByte.cu.o] Error 1
/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bf6a_00000000-5_THCTensorMathPairwise.cpp4.ii".
CMake Error at THC_generated_THCTensorMathPairwise.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir//./THC_generated_THCTensorMathPairwise.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/./THC_generated_THCTensorMathPairwise.cu.o] Error 1
/home/srthu2/CNN_tools/torch/extra/cutorch/lib/THC/THCReduce.cuh(310): error: more than one instance of overloaded function "min" matches the argument list:
            function "min(int, int)"
            function "min(unsigned int, unsigned int)"
            function "min(int, unsigned int)"
            function "min(unsigned int, int)"
            function "min(long long, long long)"
            function "min(unsigned long long, unsigned long long)"
            function "min(long long, unsigned long long)"
            function "min(unsigned long long, long long)"
            function "min(float, float)"
            function "min(double, double)"
            function "min(float, double)"
            function "min(double, float)"
            argument types are: (unsigned long, unsigned long)

1 error detected in the compilation of "/tmp/tmpxft_0000bfb4_00000000-5_THCTensorMathScan.cpp4.ii".
CMake Error at THC_generated_THCTensorMathScan.cu.o.cmake:267 (message):
  Error generating file
  /home/srthu2/CNN_tools/torch/extra/cutorch/build/lib/THC/CMakeFiles/THC.dir//./THC_generated_THCTensorMathScan.cu.o


make[2]: *** [lib/THC/CMakeFiles/THC.dir/./THC_generated_THCTensorMathScan.cu.o] Error 1

**Comment by sidharthms** [comment]
Any update on this issue?

**Comment by VictorSigma** [comment]
I too am facing the same issue. I was able to install the older version using
`luarocks install cutorch 1.0-0`

**Comment by sxjzwq** [comment]
I met the same error. Any update? I can install cutorch 1.0-0 and cunn 1.0-0, but fail to pass the test.

**Comment by VictorSigma** [comment]
Also updating to Ubuntu 16.04 fixed the issue as well. This seems to be only effecting 14.04

**Comment by AtlantixJJ** [comment]
emm....I rolled back torch repository for a few commits and everything seems to be fine.
commit 497e8b98596eebc7d2b7852d3bdb3df3a43bbe9b

**Comment by aisebastian** [comment]
I can confirm it is still happening on latest head
